docs/source/contributing/contributing-guide.rst

```diff
@@ -0,0 +1,61 @@
+************************
+Contributing Guide
+************************
+
+Thanks for checking out the contributing guide. Substra warmly welcomes contributions!
+
+Ground rules & expectations
+===========================
+
+* Be kind and thoughtful in your conversations around this project. We all come from different backgrounds and projects, which means we likely have different perspectives on how things should be done. Try to listen to others rather than convince them that your way is correct.
+* Substra has a :doc:`Contributor Code of Conduct </contributing/code-of-conduct>`. By participating in this project, you agree to abide by its terms.
+
+Who are contributors ?
+======================
+
+Contributors are any person that have contributed to the code. It does not matter whether it's a typo fix or 10k lines of code. Making a contribution however does not automatically entitle you to copyright over that code. For copyright the contribution must be significant enough meet the `threshold of originality <https://en.wikipedia.org/wiki/Threshold_of_originality>`, which basically means that your code is somewhat unique and non-generic. Fixing a typo does not give you access to copyright over that word or sentence.
```

Comment:
link is broken
<img width="742" alt="image" src="https://user-images.githubusercontent.com/5354301/210574255-0d5120c6-b0dc-4fbc-a614-76df5df46fbd.png">